src/renovate.ts

```diff
@@ -112,6 +116,27 @@ class Renovate {
       );
     }
   }
+
+  private validateDockerCmdFileArgument(): void {
+    const dockerCmdFile = this.input.getDockerCmdFile();
+    if (dockerCmdFile !== null) {
+      if (
+        !fs.existsSync(dockerCmdFile) ||
+        !fs.statSync(dockerCmdFile).isFile()
+      ) {
+        throw new Error(
+          `dockerCmdFile '${dockerCmdFile}' MUST be an existing file`,
+        );
+      }
+      try {
+        fs.accessSync(dockerCmdFile, fs.constants.R_OK | fs.constants.X_OK);
```

Comment:
you can reuse the stat return value:
https://github.com/containerbase/base/blob/30eddf2136736bef468fdbb9e603672640a10571/src/cli/services/path.service.ts#L166

Reply:
implemented this